ecosystem/typescript/sdk/examples/typescript/multisig_managed_coin.ts

```diff
@@ -0,0 +1,286 @@
+/* eslint-disable no-console */
+
+import dotenv from "dotenv";
+dotenv.config();
+
+import { AptosClient, AptosAccount, FaucetClient, BCS, TxnBuilderTypes, Types, HexString, FungibleAssetClient, Provider } from "aptos";
+import { sha3_256 as sha3Hash } from "@noble/hashes/sha3";
+import { FAUCET_URL, NODE_URL, fungibleStore } from "./common";
+import assert from "assert";
+
+const { AccountAddress, EntryFunction, MultiSig, MultiSigTransactionPayload, TransactionPayloadMultisig } =
+    TxnBuilderTypes;
+
+// Step 0: After publishing the example code with the example code under any `MODULE_ADDR` using CLI or any sdk.
+const MODULE_ADDR = process.env.MODULE_ADDR;
+const ASSET_SYMBOL = "MEME";
+
+/**
+ * This code example demonstrates how to use framework mulltisig account module to manage fungible asset with exmaple move code.
+ */
+(async () => {
+    const client = new Provider({ fullnodeUrl: NODE_URL });
```

Comment:
add `Network` to `import {...} from "aptos"` and then
```suggestion
    const client = new Provider(Network.DEVNET);
```